tools/lib/provision.py

```diff
@@ -357,7 +356,7 @@ def main(options):
 
         for path in paths:
             with open(path, 'r') as file_to_hash:
-                sha1sum.update(force_bytes(file_to_hash.read()))
+                sha1sum.update(file_to_hash.read().encode('utf8'))
```

Comment:
A slightly cleaner version of this code would be to say `open(path, 'rb')` (with the `b`), and then `read()` will return bytes in the first place.